css/style.css

```diff
@@ -6,14 +6,207 @@ body {
+    height: 3em;   
+    border: none;
+    border-radius: 5%;
+    font-size: calc(0.75em + 1vw);
+    padding: 5px 10px;
+    margin-top: 30px;
+}
+/*Main-Container style*/
+.container{
+    margin-top:75px ;
+}
+ 
+.container-heading p{
+    font-size:calc(1.5em + 1vw);
+    text-align: center;
+}
+.container-icons{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-wrap: wrap;
+    
+}
+.container-icons-wrapper{
+    display:flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+}
+.container-icons-wrapper img{
+    width: 7.5em;
+    margin:2em;    
+}
+.container-icons-wrapper p{
+    font-size: 3em;
+    font-size:calc(1em + 0.75vw);
+    margin:2em;
+    text-align:center;
+  }
+
+
+/*Footer style*/
+.footer-icons img{
+width:1.5em;
+margin-left:10px;
+margin-right:10px;
+}
+footer{
+   display:flex;
+   flex-direction: column;
+   justify-content: center;
+   align-items: center;
+ }
+.footer-heading p, .footer-copyright p{
+    font-size: 1.5em;
+    font-size:calc(0.8em + 0.5vw);
+}
+/* adding media query-inorder not to show navbar for big screens,make visible the mobile nav bar and changing background image height */
+@media screen and (max-width:1500px){
+    nav .big-screen{
+        display: none; 
+    }
+
+    .mobile-screen {
+        display:block;
+      }
+    .container-icons{
+        flex-direction: column;
+     
+    }
+    .hero{
+        text-align: center;
+        background-image: url('../img/first-background.jpg');
+        height: 500px;
+        background-repeat: no-repeat;
+        background-position: center;
+        background-size: cover;
+        color:white;
+        display:flex;
+        flex-direction: column;
+        justify-content: center;    
+    }
+    .topnav {
+        overflow: hidden;
+        background-color: rgb(94, 91, 91);
+        position: relative;
+        display:none;
+        min-width: 100vw;
+        margin-left:0;
+      }
+      
+      .topnav #myLinks {
+        display: none;
+        
+      }
+
+
+      .topnav li{
+          list-style-type: none;
+      }
+      .topnav a {
+        color: white;
+        padding: 14px 16px;
+        text-decoration: none;
+        font-size: 17px;
+        display: block;
+      }
+      
+      .topnav a.icon {
+        background: rgb(151, 146, 146);
+        display: block;
+        position: absolute;
+        right: 0;
+        top: 0;
+      }
+      
+      .topnav a:hover {
+        background-color: #ddd;
+        color: black;
+      }
+      
+      .active {
+        background-color: #4CAF50;
+        color: white;
+      }
+    header{
+     
+        margin: 0px 0px;
+    }
+    .logo{
+     
+        padding-left:20px;
+        
+    }
+    #start-link{
+        margin-right: 20px;
+    }
+
+}
 
+/* adding media query for big screens-which hide the mobile nav bar and links */
+@media screen and (min-width:1500px){
```

Comment:
Nice - I suggest reducing your breakpoint as 1500px is already considered a large desktop. Try breaking somewhere between 600px and 800px instead (a common one if 768px because that is the width of an iPad screen)

Reply:
Thank you very much,  I have changed it to 768px

@media screen and (min-width:768px){